scripts/helper_functions.sh

```diff
@@ -120,7 +120,7 @@ Log() {
 DiscordMessage() {
   local message="$1"
   local level="$2"
-  if [ -n "$level" ]; then
+  if [ -z "$level" ]; then
```

Comment:
Good catch

Reply:
I was testing something with webhooks and thats where I noticed it. Small but crucial line 😊